packages/android_alarm_manager_plus/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: android_alarm_manager_plus
 description: Flutter plugin for accessing the Android AlarmManager service, and
   running Dart code in the background when alarms fire.
-version: 2.0.7+1
+version: 2.0.8
```

Comment:
Same comment as in `CHANGELOG` about not needing a new version.

Reply:
Thanks for that! This has been removed also